bindings/protocols/coap/mapping.ttl

```diff
@@ -0,0 +1,108 @@
+@prefix : <https://www.w3.org/ns/wot/binding/coap-validation#> .
+@prefix td: <https://www.w3.org/ns/wot/td#> .
+@prefix wotsec: <https://www.w3.org/2019/wot/security#> .
+@prefix dc: <http://purl.org/dc/elements/1.1/> .
+@prefix hctl: <https://www.w3.org/2019/wot/hypermedia#> .
+@prefix vs: <http://www.w3.org/2003/06/sw-vocab-status/ns#> .
+@prefix owl: <http://www.w3.org/2002/07/owl#> .
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix xml: <http://www.w3.org/XML/1998/namespace> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+@prefix foaf: <http://xmlns.com/foaf/0.1/> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix skos: <http://www.w3.org/2004/02/skos/core#> .
+@prefix vann: <http://purl.org/vocab/vann/> .
+@prefix dcterms: <http://purl.org/dc/terms/> .
+@prefix schema: <http://schema.org/> .
+@prefix sh: <http://www.w3.org/ns/shacl#> .
+@prefix cov: <https://www.w3.org/2019/wot/coap#> .
+@prefix binding: <https://www.w3.org/2019/wot/binding#> .
+@base <https://www.w3.org/ns/wot/binding/coap-validation#> .
+
+<> rdf:type owl:Ontology ;
+    owl:imports <https://www.w3.org/2019/wot/coap> ;
+    vann:preferredNamespacePrefix "cov" ;
+    vann:preferredNamespaceUri : ;
+    rdfs:comment "Vocabulary to map CoAP concepts to the standard Thing Interaction model."@en ;
+    rdfs:label "CoAP protocol binding"@en .
+
+:CoapFormShape a sh:NodeShape ;
+    sh:targetClass hctl:Form ;
+	sh:order 1 ; # Used to select the shape
+    sh:closed false ;
+    sh:property [
+        sh:path cov:hasMethod ;
+        sh:datatype xsd:string ;
+        sh:maxCount 1 ;
+    ] ;
+    sh:property [
+        sh:path cov:hasBlockwise ;
+        sh:datatype cov:BlockWiseTransferParameters ;
+        sh:maxCount 1 ;
+    ] ;
+    sh:property [
+        sh:path cov:hasQBlockwise ;
+        sh:datatype cov:BlockWiseTransferParameters ;
```

Comment:
```suggestion
        sh:class cov:BlockWiseTransferParameters ;
```

Reply:
I might be good to have some "unit tests" in the form of some TDs that get validated against these SHACL shapes as part of CI/CD.